src/components/Calendar/index.tsx

```diff
@@ -233,6 +234,34 @@ export default function Calendar({
     }
   });
 
+  const filteredCourses = oscar.courses.filter((course) => {
+    if (desiredCourses.includes(course.id)) {
+      return course;
+    }
+    return '';
+  });
+
+  const finalFilteredCourses = filteredCourses.filter((course) => {
+    let match;
+
+    pinnedCrns.forEach((courseCrn) => {
+      course.sections.forEach((section) => {
+        if (courseCrn === section.crn) {
+          if (
+            section.campus !== 'Georgia Tech-Atlanta *' ||
+            section.meetings[0]?.days.includes('S')
+          ) {
```

Comment:
Campus shouldn't be used as a filter. We want to filter sections whose period is undefined (i.e. "TBA") since only TBA sections (aside from weekend ones) are hidden from view.